packages/@react-spectrum/textfield/test/TextArea.test.js

```diff
@@ -47,11 +47,31 @@ describe('TextArea', () => {
     ${'v3 TextArea'}    | ${TextArea}      | ${{isQuiet: true, onChange}}
     ${'v2 TextArea'}    | ${V2TextArea}    | ${{quiet: true, onChange}}
   `('$Name quiet variant automatically adjusts its vertical height on change', ({Component, props}) => {
-    let tree = renderComponent(Component, props);
-    let input = tree.getByTestId(testId);
+    act(() => {
+      renderComponent(Component, props);
+    });
 
+    let input = screen.getByTestId(testId);
```

Comment:
learned something new, hadn't seen screen before
is there a reason you are using it over the returned values from render?

Reply:
I will redo the test in the later commit and try to do it the right way 🤣 🤣